modules/dynamic-plugins/con-understand-why-wiring-is-required-for-plugin-deployment.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: CONCEPT
+[id="con-understand-why-wiring-is-required-for-plugin-deployment_{context}"]
+
+= Understand why wiring is required for plugin deployment
```

Comment:
I would remove 'development'. It makes it sound that this is needed only for development, which is not right.
Wiring configuration is required for running frontend plugin. Regardless if it is development or production run. 

```suggestion
= Understand why wiring is required
```